### ISSUE TITLE
casacore21/casacore2 conflict for python-casacore on Ubuntu 14.04

### ISSUE THREAD

**Issue by sjperkins** [ska-sa/montblanc#172]
python-casacore depends on casacore2 but if you've got casacore21 installed it finds that and complains.

Solution is to just install casacore2

**Comment by JSKenyon** [comment]
I encountered this error whilst attempting a fresh install of MB on my 14.04 desktop machine. I followed the installation instructions but found that there was a conflict between my installed python-casacore and in prerequisite packages of MB. After removing the existing python-casacore, I was able to install the prerequisite packages, but was then unable to pip install python-casacore into my virtualenv. Again, there was some sort of conflict.

Having fought with the problem for a while, I think the problem is that python-casacore is built against casacore2 while the prerequisite packages call for casacore21. I solved the issue by removing casacore21 and instead installing casacore2. This allowed me to apt-get install python-casacore without running into the errors I had previously. It is still unclear whether or not this fix will cause unexpected behaviour. 

Having contacted @gijzelaerr, it appears that the 16.04 packages should not have this problem, as they are all correctly built against casacore21.
